### PR TITLE
Add e2e tests for native query drills

### PR DESCRIPTION
**Pull request by ranquild**
Epic https://github.com/metabase/metabase/issues/48466
Testing plan https://github.com/metabase/metabase/issues/48469

### CODE REVIEW

File: e2e/test/scenarios/question/native-query-drill.cy.spec.ts [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+  left,
+  right,
+  bottom,
+}: {
+  top: number;
+  left: number;
+  right: number;
+  bottom: number;
+}) {
+  cy.findByTestId("visualization-root")
```
Comment (uladzimirdev) [inline e2e/test/scenarios/question/native-query-drill.cy.spec.ts:471]: FYI https://github.com/metabase/metabase/blob/master/e2e/test/scenarios/question/saved.cy.spec.js#L266-L270

this is the way of testing D&D recommended by Cypress

File: e2e/test/scenarios/question/native-query-drill.cy.spec.ts [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+        visitQuestion: true,
+      });
+      assertQueryBuilderRowCount(10);
+      applyBrushFilter({ left: 200, right: 800 });
+      cy.wait("@dataset");
+      assertQueryBuilderRowCount(5);
+    });
+
+    it("coordinates filter", () => {
+      cy.log("pin map");
```
Comment (uladzimirdev) [inline e2e/test/scenarios/question/native-query-drill.cy.spec.ts:374]: we have at least `should apply brush filters to the native query series selecting area range when axis is a number` in `e2e/test/scenarios/visualizations-charts/line_chart.cy.spec.js`, let's keep only one

**Review by uladzimirdev — APPROVED** [review]
I like the idea of the separated file overall, but we have already some tests for drills in other files, it's not a bad thing that we test the same functionality several times, but some way of avoiding repeating should be made.

Also we have this [PR in progress](https://github.com/metabase/metabase/pull/48289) from @kamilmielnik , I expect some overlap as well

**Comment by kamilmielnik** [comment]
> Also we have this [PR in progress](https://github.com/metabase/metabase/pull/48289) from @kamilmielnik , I expect some overlap as well

The PR I'm working on tests drilling via dashcard headers on non-native queries.

**Pull request merged**